### PR TITLE
My first check  project for code review practice.

### PR DESCRIPTION
**Pull request by bertinkambale**
In this project, I have set up an HTML file and CSS file and READ.me file  so that I may review them

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+            <nav>
+                <ul>
+                    <li><a href="">Product</a></li>
+                    <li><a href="">Entreprises</a></li>
+                    <li><a href="">Freetrials</a></li>
+                    <li><a href="">Precing</a></li>
+                    <li><a href="">Support&Learning</a></li>
+                    <li><a href="">About Us</a></li>
+                </ul>
+            </nav>
```
Comment (bertinkambale) [inline index.html:23]: please kindly remove the nav tag, it is not used in the code.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+   <!--3rd block-->
+   
+            <section class="lastblock">
+                    <div>
+                        <h1>DevCraft</h1>
+                        <p>NET UI controls reporting and developerproductivity tools</p>
+                        <button class="l000">learn More</button>
+                    </div>
+
+                    <div>
```
Comment (bertinkambale) [inline index.html:44]: kindly remove spaces in your HTML codes

File: style.css [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+    text-decoration: none;
+    text-transform: uppercase;
+}
+.header1 a:hover{
+    background-color: white;
+    color:black;
+    padding:5px 10px;
+}
+
+
```
Comment (bertinkambale) [inline style.css:38]: please remove that space

**Review by bertinkambale — COMMENTED** [review]
#changes required.

Please kindly check my comment below and make all the required.
please don't forget to remove those spaces between div elements, and in CSS to space after closing a class.
kindly remove unused pictures in your repository.
submit for a review when you are done.
thanks.

**Review by bertinkambale — COMMENTED** [review]
please kindly use Linters to check your code if is working properly with errors.

**Review by tresorsawasawa — CHANGES_REQUESTED** [review]
# STATUS: REQUEST CHANGES :recycle::recycle: 

Hi @bertinkambale 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] I think it would be better if you add a good summary after you completed the review by adding gifs to make it more friendly, **highlight** some efforts improved by the student, etc. You can use one template of your previous received reviews. 
- [ ] **Check the comments under the review**.

## Optional suggestions
- [ **OPTIONAL** ] I think it would be better to fix linters errors for helping you apply best practices in your code as they check errors. I know they are not required in the projects but I suggest you do so to improve your code.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @bertinkambale ,  

Your project is complete! Great job for making the changes requested by the previous reviewer 🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request
- Validations correctly implemented
- No linter errors

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**